discover/deployment/articles-dxp/03-maintaining-liferay/01-patching-liferay/01-patch-types.markdown

```diff
@@ -0,0 +1,46 @@
+# Patch types [](id=patch-types)
+
+Liferay ships Digital Enterprise fixes through three different channels. Fix packs and hotfixes. You can read about the differences in this document.
```

Comment:
I forgot to save what I added before commit and the service packs were never added. I've pushed some changes to my branch. Could you please take it as the base for the review?